File: python/268.MissingNumber.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        sum = 0
        nums_len = len(nums)
        for ele in nums:
        	sum += ele
        return nums_len*(nums_len+1)/2-sum

File: cpp/268.MissingNumber.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
    	int len = nums.size();
    	int sum = 0;
    	for (int i=0;i<len;++i) sum += nums[i];
    	return (len*(len+1)>>1) - sum;
    }
};